docs/team/bentanjunrong.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Benjamin Tan's Project Portfolio Page
+---
+
+### Project: NUS ModBook
+
+NUS ModBook is a desktop application for NUS students to manage modules, optimized for use via a Command Line Interface.
+For this project, I was the Code Quality Lead.
+
+Given below are my contributions to the project.
+
+* **Code contributed**: [RepoSense](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=bentanjunrong&tabRepo=AY2122S1-CS2103T-T13-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&zFR=false)
+
+* **New Features**:
+    * Added the representation for `Lessons` in the ModBook [#53](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/53)
+        * Created `Lesson`, `LessonName` and `Day` classes, along with all fields and methods for them
+        * Added methods in ParserUtil to parse Lessons, LessonNames and Days
+        * Added tests for Lesson, LessonName, and Day and their respective ParserUtil methods
+    * Added functionality to **edit** Modules, Exams, Lessons [#101](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/101)
+        * Added `edit` commands (for Modules, Exams, and Lessons) and corresponding command parsers
+        * Added `EditModDescriptor`, `EditExamDescriptor` and `EditLessonDescriptor` classes to store the details of each edit command
+        * Added tests for all relevant classes and methods added
+        * Added utility methods to recreate command strings based on the `EditModDescriptor`, `EditExamDescriptor` and `EditLessonDescriptor`
+
+* **Enhancements to existing features**:
+    * Added flexible constructors and readable string for the Day enum [#131](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/131)
+  * Adapt `Model` interface and `ModelManager` class [#60](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/60)
+      * Added relevant CRUD methods for modules in the `ModelManager` and `ModBook` classes
+      * Implemented `UniqueModuleList` class to handle list operations for modules
+      
```

Comment:
can standardise the bullet indentations here

Reply:
i can, but i don't want to